crates/matrix-sdk-crypto/src/verification/cache.rs

```diff
@@ -183,15 +194,24 @@ impl VerificationCache {
         recipient: &UserId,
         recipient_device: &DeviceId,
         content: OutgoingContent,
+        request_info: Option<RequestInfo>,
     ) {
+        let request_id = if let Some(request_info) = request_info {
+            trace!(%recipient, ?request_info, "Putting flow id waiting for confirmation");
```

Comment:
There seem to be two verbs in this log message that don't compose, at least in my head (putting, waiting). Is it just me?

Reply:
Indeed, that's a derpy sentence. Fixed in https://github.com/matrix-org/matrix-rust-sdk/pull/1145/commits/3fe3785dcb88c6fc9103e64f772f5f5b1acca96e.